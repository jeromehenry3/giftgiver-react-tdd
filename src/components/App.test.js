import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

const app = shallow(<App />);

it('rendres correctly', () => {
    expect(app).toMatchSnapshot();
});