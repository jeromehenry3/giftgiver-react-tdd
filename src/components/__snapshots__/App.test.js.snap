// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
"<div>
  <h2>
    Gift Giver
  </h2>
  <Button className=\\"btn-add\\" onClick={[Function]} variant=\\"primary\\" active={false} disabled={false} type=\\"button\\">
    Ajouter un cadeau
  </Button>
</div>"
`;
